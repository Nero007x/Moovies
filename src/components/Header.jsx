import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const Header = () => {
  return (
    <View style={styles.header}>
      <Text style={styles.headerTitle}>Movies List</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  header: {
    height: 60,
    paddingTop: 15,
    backgroundColor: '#2196F3',
    justifyContent: 'center',
    alignItems: 'center',
  },
  headerTitle: {
    color: 'white',
    fontSize: 20,
    fontWeight: 'bold',
  },
});

export default Header;
