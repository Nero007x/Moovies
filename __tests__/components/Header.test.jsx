import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../src/components/Header';

test('Header component renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
