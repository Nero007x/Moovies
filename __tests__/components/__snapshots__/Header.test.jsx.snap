// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#2196F3",
      "height": 60,
      "justifyContent": "center",
      "paddingTop": 15,
    }
  }
>
  <Text
    style={
      {
        "color": "white",
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    Movies List
  </Text>
</View>
`;
